README.md

```diff
@@ -88,14 +88,21 @@ python makeScanInput.py -d Summer16v3Fast -r input/dict_scan_2016.py -w input/in
 python makeScanInput.py -d Fall17Fast -r input/dict_scan_2017.py -w input/input_sets_scan_2017.txt -e batch/exportScan2017.sh -y 2017 -n 10
 python makeScanInput.py -d Autumn18Fast -r input/dict_scan_2018.py -w input/input_sets_scan_2018.txt -e batch/exportScan2018.sh -y 2018 -n 10
 ```
-The last argument splits the scan input list into multiple blocks (each containing `n` ntuple files) for batch submission.  
+The last argument splits the scan input list into multiple blocks (each containing `n` ntuple files) for batch submission. In order to run the FullSim signal scans you will need to remove the word `Fast` from the eras and make the following file replacements:
```

Comment:
by "eras" you mean the `-d` arguments? I would just say the `-d` argument directly, as it corresponds to the TreeMaker folder name, not the era/scenario

Reply:
Changed